### ISSUE TITLE
BB10 app signing in 2024

### ISSUE THREAD

**Issue by ChomperT** [BerryFarm/Term49#7]
Hello, may I ask if it's still possible to sign BB10 apps now? Is it possible to do so using an expired bbidtoken.csk file?

**Comment by xplanthris** [comment]
No, if you have a debug token you can set the time back to use it.

**Comment by robertedward883** [comment]
Theoretically, you can edit the fields in the file manually or set up the date before its expiration and disable some services in Windows, which are checking the date in the network or something like that. Maybe you can share your token and pass?
